src/db/z_db.erl

```diff
@@ -765,18 +777,20 @@ table_exists(Table, Context) ->
 
 
 %% @doc Make sure that a table is dropped, only when the table exists
+-spec drop_table(table_name(), #context{}) -> ok.
 drop_table(Name, Context) when is_atom(Name) ->
     drop_table(atom_to_list(Name), Context);
 drop_table(Name, Context) ->
     case table_exists(Name, Context) of
-        true -> q("drop table \""++Name++"\"", Context);
+        true -> q("drop table \""++Name++"\"", Context), ok;
         false -> ok
     end.
 
 
 %% @doc Ensure that a table with the given columns exists, alter any existing table
 %% to add, modify or drop columns.  The 'id' (with type serial) column _must_ be defined
```

Comment:
Line 791 is too long: %% to add, modify or drop columns.  The 'id' (with type serial) column _must_ be defined.